src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -105,7 +106,7 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
       message={item?.message}
       attachments={item?.attachmentJson}
       time={calculateTime(item?.updatedAt, true)}
-      username={handleUserName(item)}
+      username={handleUserName(item, selectedChannel)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 93, the `handleUserName` function is now being called with two parameters: `item` and `selectedChannel`. Ensure that the updated function definition supports these parameters and behaves as expected.
```diff
-      username={handleUserName(item)}
+      username={handleUserName(item, selectedChannel)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->